packages/ngtools/webpack/src/transformers/remove-ivy-jit-support-calls_spec.ts

```diff
@@ -164,5 +164,53 @@ describe('@ngtools/webpack transformers', () => {
       expect(tags.oneLine`${result}`).toEqual(tags.oneLine`${output}`);
     });
 
+    it(`should strip unused imports for 'ExpressionWithTypeArguments' when parent token is 'ImplementsKeyword`, () => {
```

Comment:
Can a general elide imports test be added as well (or instead if preferred) since this can technically happen any time the import eliding transformer is run regardless of Ivy usage?

Reply:
Yeah it can. I’ll add a test tomorrow.